curriculum/README.md

```diff
@@ -304,4 +304,4 @@ on the TEALS dashboard, or to [APSquad@tealsk12.org](mailto:APSquad@tealsk12.org
 [Timing and Pacing]: https://www.youtube.com/watch?v=LkGh03UZ724
 [Twiddla]: http://www.twiddla.com
 [Vyew]: http://www.vyew.com
-[here]: https://1drv.ms/w/s!AmOr6JmWN27GgfZLyI6QVmMVfVj2Nw
+[here]: apcsa-public/curriculum/AP CS A Course Audit Instructions.docx 
```

Comment:
This link won't work for a couple of reasons.

First, the release it built up from multiple sources, and since you've put this document at the virtual root of the release (`apcsa-public\curriculum`), the path would actually look like `./AP CS A Course Audit Instructions.pdf` (possibly with spaces encoded as `%20`). The `apcsa-public/curriculum` path only exists in the curriculum sources. You can try to build the release yourself by going to the `apcsa-instructor` repo and in the command line running `make clean build`. Kenney had issues with the Microsoft corporate version of Word; I don't know if this will work for you.

Second, local Markdown (via `grip`) doesn't understand MIME types, so doesn't know how to load .pdf and .docx files. For those browsing the curriculum content from GitBook, this document won't exist, since it wasn't referenced in the `SUMMARY.md` file.